tools/pipesys/src/cmd/mod.rs

```diff
@@ -0,0 +1,117 @@
+#[cfg_attr(target_os = "linux", path = "link.rs")]
+#[cfg_attr(not(target_os = "linux"), path = "non_linux_link.rs")]
+mod link;
+
+use self::link::Link;
+use pipesys::server::Server as Serve;
+
+use anyhow::Result;
+#[cfg(target_os = "linux")]
+use anyhow::{ensure, Context};
+use clap::Parser;
+use env_logger::Builder;
```

Comment:
nit: I think it'd be slightly more readable not to `use` this since it's such a generic name